dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -18,11 +18,9 @@
        With the default strategy of merge, the user may choose between a temporary
        table and view (defaulting to view).
 
-       The append strategy can use a view because it will run a single INSERT statement
-       when the unique_key is none.
-
```

Comment:
@McKnight-42 I think we deleted a bit too much here.

We should keep this:
```
       The append strategy can use a view because it will run a single INSERT statement
```

But make sure this doesn't sneak back in:
```
       when the unique_key is none.
```